python/tvm/relay/frontend/pytorch.py

```diff
@@ -902,6 +902,21 @@ def _report_missing_conversion(op_names):
         msg = "The following operators are not implemented: {}".format(missing)
         raise NotImplementedError(msg)
 
+def _check_input_names(graph, input_shapes):
+    """ Check the graph inputs match the inputs """
+    # remove self at the 0th arg
+    ir_inputs = _get_input_names(graph)[1:]
```

Comment:
Passing script_module and using _get_graph_input_names would be simpler?

Reply:
Fair enough - though I was just trying to avoid the extra graph.copy().